### ISSUE TITLE
[FEATURE]: Deduplication Tool

### ISSUE THREAD

**Issue by Saijin-Naib** [marius-wieschollek/passwords#611]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **feature request** for _one_ feature, not a question, discussion or multiple features.
- [X] This is not a feature request for the [browser extension](https://github.com/marius-wieschollek/passwords-webextension) or another client.
- [X] This feature is **not** already requested on Github _(I've searched it)_.
- [X] This feature does not already exist _(I checked the [wiki](https://git.mdns.eu/nextcloud/passwords/-/wikis/home))_.

### Current Status

Currently, there is no way to manage duplicate password entries. If you have moved browsers or managers over the years (or you are old enough that your passwords list predates synced password tools), you will have duplicates. 

It seems like the Bulk actions work might need to be merged first, in order to provide some of the needed features and UX. 

### Feature Description

1. Duplicates are shown in a special folder, stacked or grouped together in the list
2. User can choose which duplicates for a given group to merge together, and which is the primary (its existing fields don't get updated)
3. Once flattened, passwords are no longer shown in the Duplicates folder

### Additional Context

Similar to how BitWarden handles it.

**Comment by marius-wieschollek** [comment]
Passwords checks for duplicates by default.
You can find them under "Security" > "Weak".

Clicking on the yellow shield icon of any duplicate password will trigger a search for all entries with the same password.

There are some instances where a duplicate isn't shown in the search:
- If it's already in the trash. Check the trash bin and empty it.
- If it has recently been deleted. The status will update within one day.
- If the duplicate itself is hidden. You can use the "Recover hidden items" action in the settings to unhide any hidden password.

**Comment by Saijin-Naib** [comment]
This list is not just a list of duplicate password entries as many might think, such as multiple imports, which would then share the same username/URI/password, which is what I would like to work on clearing, instead of picking through other items that might just have a shared password but aren't actually duplicates from importing.

**Comment by marius-wieschollek** [comment]
Ah, ok. So this is about duplicates in terms of username+website or other properties.

**Comment by Saijin-Naib** [comment]
Yes, exactly! Perhaps the yellow category could have a way to switch the filter rules to show things like duplicate password, uri, etc
